src/components/ProductDetail.jsx

```diff
@@ -0,0 +1,42 @@
+import { useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import { useFetch } from '../hooks/useFetch';
+import Box from './Box';
+import Text from './Text';
+import Image from './Image';
+import Button from './Button';
+
+function ProductDetails() {
```

Comment:
It's okay to help each other but you and Sebastian have identical code for this file. Make sure that the work is your own and you have a solid understanding on how this works.